hack/dev-models/gke-vllm-gpu-adapters.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kubeai.org/v1
+kind: Model
+metadata:
+  name: dev
+  annotations:
+    # Have the controller send requests to localhost to allow for
+    # running the controller locally (assuming a port-forward is in place).
+    model-pod-ip: "127.0.0.1"
+    model-pod-port: "7000" 
+spec:
+  features: [TextGeneration]
+  owner: meta-llama
+  url: hf://TinyLlama/TinyLlama-1.1B-Chat-v0.3
+  #url: hf://meta-llama/Llama-2-7b
+  adapters:
+  - id: test
+    url: hf://jashing/tinyllama-colorist-lora
```

Comment:
does vLLM support directly loading this adapter from HF or is it a hard requirement to download the lora adapter first?

Reply:
vLLM can load it from HF but not S3

Reply:
See: https://github.com/vllm-project/vllm/blob/d1c6799b8870e513bf4f2305cbf6cda9fc3d773b/vllm/lora/utils.py#L178